src/components/WhaleSightingViewer/WhaleSightingViewer.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../App.scss";
+
+* {
```

Comment:
I'm quite uneasy of rules that target everything! Shouldn't the font already be set up at the top level?